docs/help/index.md

```diff
@@ -57,7 +45,7 @@ Open the [diagram] for ground G-2 (column G, row 2) in the [Whiting catalogue].
 
 ### Highlight a thread
 
-In the thread diagram on the far right, click on the gray square at the start of a thread.  You may need to scroll to the top of the diagram to see it.  The colour of the thread will change from black to red.  Click on the square again and the thread colour returns to black.  You can highlight more than one thread at a time.  For more colours and options, visit [thread properties](Thread-Properties).
+In the thread diagram on the far right, click on the gray square at the start of a thread or the black bobbin at the end of a thread.  You may need to scroll the diagram to see the ends of a thread.  The colour of the thread will change from black to red.  Click on the square or bobbin again and the thread colour returns to black.  You can highlight more than one thread at a time.  For more colours and options, visit [thread properties](Thread-Properties).
 
 ![](images/G2-toggle-thread.png)
```

Comment:
The image is too large fit on a reasonable monitor together with the text. It requires too much scrolling to understand.

Reply:
I will resize.

Reply:
I didn't mean plain resizing. These two images show four purposes. The handles, the pop-ups, thread behavior inside the pattern/stitch and along the edge. The bug at the bottom with the bobbins two by two in the wrong order might be confusing.

![Screenshot 2020-09-13 at 08 36 51](https://user-images.githubusercontent.com/10553630/93012035-ff96d600-f59c-11ea-985a-1424fc0c0cd0.png)

Reply:
Now that I study the top of these diagrams: show one or two rows and columns more and two diagram variants. One variant with two twists between the stitches and one with a single twist. I'll leave it up to you to stick to two diagrams or make it four. plenty of room to have four side by side. It shows how the threads change roles at the edge or not depending on an odd or even number of twists. Roles as in inside/outside-threads. Note that you can play with the zoom factor of your browser to play with the ration of the pop-up size and handle-size. At least in firefox the pop-up doesn't increase along.

Reply:
I have read over the tutorial again and I have to disagree with changing to a simpler ground.  The entire tutorial is about playing with G2 from Whiting's sampler. A person following along with the tutorial will expect to see G2, not Brussel's ground.

Reply:
One option is to re-write this whole "Getting started" tutorial with a simpler example.  This might not be a bad idea.  I was trying to keep the tutorial close to the original Binche Kompakt 31 example.  However, now that Binche Kompakt 31 has a home of its own, a simple example may be more useful as a starting point.